sources/src/App.vue

```diff
@@ -665,7 +665,7 @@ export default {
             text: {
               header: "Tools",
               text:
-                '<p> Supporting tools ease everything surrounding tests: editing, running, building and so on. Most of these tools are developed as separate projects, but some are built into the framework itself. </p> <p> <a href="https://github.com/robotframework/robotframework.github.com" target="_blank">Let us know</a> if there are useful tools missing from the list. </p>'
+                '<p> Supporting tools ease everything surrounding automation and tests: editing, running, building and so on. Most of these tools are developed as separate projects, but some are built into the framework itself. </p> <p> <a href="https://github.com/robotframework/robotframework.github.com" target="_blank">Let us know</a> if useful tools are missing from the list. </p>'
```

Comment:
Should "surrounding automation and tests" be changed just to "surrounding automation"? I don't particularly like that "surrounding" word either...